apps/jehovakel_ex_event_store/lib/event_store/link_appendable_events.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Shared.LinkAppendableEvents do
+  use GenServer
```

Comment:
Das ist so ein typischer Kanditat, den man vergisst mit in seinen Supervisor aufzunehmen. Und auch die Strategie dafür ist aus der App heraus evtl. unklar (Sollte im Fehlerfall wahrscheinlich mit dem EventStore synchron neugestartet werden).
Evtl. könnte ein eigener Supervisor mitgeliefert werden, der beides überwacht.

Reply:
Man könnte natürlich beim initialisieren des EventStores sowas machen, wie: 
`use Shared.EventStore, link_events: true` oder so, um das Linking einzuschalten.
EventStore sterben lassen, weil Links nicht gemacht werden, halte ich für problematisch. Andersrum natürlich mega sinnvoll.
Ich würde ein wenig damit warten. Habe das noch nicht komplett im Kopf, wie das am besten sein soll...